### ISSUE TITLE
Types change when editing connection

### ISSUE THREAD

**Issue by samlevy3** [UDub-Impact/OData-Connector#34]
Currently, you can connect to a table, edit the connection, and reconnect to a different one. However, during this reconnection, the types for some of the fields change and this causes the data to be null or the connection to fail.

**Comment by samlevy3** [comment]
This appears to be intentional behavior by google data studio. Based on this document: https://support.google.com/datastudio/answer/7648843?hl=en#zippy=%2Cin-this-article It appears that type changes are not updated when the connection is changed. We think this is why the types of the fields carry over from the previous connection causing the data types to be incorrect. We are looking into how we could circumvent this to force type changes when the connection is altered.

**Comment by samlevy3** [comment]
After further research we realized editing the connection is not recommended for this situation. It overwrites the previous connection which had been created which not only could break reports made using the previous connection but also has this issue of not updating types. We added a [section](https://github.com/UDub-Impact/OData-Connector/blob/master/connector%20guide.md#edit-connection) to the connector guide specifying that users should not edit the connection.
